tests/integration/extenders/RoutesTest.php

```diff
@@ -47,6 +47,29 @@ public function custom_route_can_be_added_by_extender()
         $this->assertEquals(200, $response->getStatusCode());
         $this->assertEquals('Hello Flarumites!', $response->getBody());
     }
+
+    /**
+     * @test
+     */
+    public function custom_route_can_override_existing_route_if_removed()
+    {
+        $this->extend(
+            (new Extend\Routes('api'))
+                ->remove('GET', 'forum.show')
+                ->get('/overridenRoute', 'forum.show', CustomRoute::class)
```

Comment:
Why not have the overriden route at the same path as the old route?

Reply:
Don't remember, I've tweaked the tests now.